drivers/pca9685/Kconfig

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_PCA9685
+    bool "PCA9685 I2C PWM controller"
+    depends on HAS_PERIPH_GPIO
+    depends on HAS_PERIPH_I2C
+    depends on TEST_KCONFIG
```

Comment:
Seem to be missing:

```
# efm32 CPU doesn't support PWM_RIGHT
FEATURES_BLACKLIST += arch_efm32
```

So I guess `depends on !CPU_ARCH_EFM32`

Reply:
Thanks! Added. I made it depend on  `!HAS_CPU_EFM32`,  because efm32 is no architecture. Alternatively the symbol `CPU_COMMON_EFM32` could be used instead.